props/saber_fett263_G_buttons.h

```diff
@@ -0,0 +1,445 @@
+// Fett263 Buttons with Gesture Controls and Battle Mode with Smart Lockup
```

Comment:
I wonder if it would have been better to just add all of this to the existing fett263 prop file and use defines to turn features on and off?

Reply:
For backwards compatible I was worried with all the new gestures it may confuse users, there are a lot of gesture additions to the G file that are related and thought too many defines would be cumbersome so having the "G" was to signify the addition of the gestures so users just pick prop and use with fewer defines.  If you prefer to reduce the number of prop files I can modify and we can just make a single with multiple defines, I am ok with either approach.

Reply:
If all the new defines default to "off", then backwards compatibility is not a problem.
From what I can tell from this file, it looks like you have most or all of those defines anyways.
Since this seems to be a natural extension of the other prop file, I think it would be better to just work on that file.
If this was a completely new, approach, with very little shared functionality, then a new file would be best.

Reply:
OK, makes sense I'll make edits and then remove this file and update existing saber_fett263_buttons.h